.github/pull_request_template.md

```diff
@@ -1,16 +1,26 @@
 ## Summary
-> Describe what the PR does and how to test. Photos and videos are recommended.
-
 
+<!--
+Describe what the PR does and how to test.
+Photos and videos are recommended.
```

Comment:
Could we keep this? Maybe add it to the title then `Summary (photos and videos recommended)`

Reply:
We are keeping this. People creating the PR will see this. It's just not rendered.

Reply:
Ah brain fart. Thanks